src/overlays/actors/ovl_En_Bba_01/z_en_bba_01.c

```diff
@@ -265,8 +265,8 @@ s32 EnBba01_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f*
         OPEN_DISPS(play->state.gfxCtx);
 
         gSPSegment(POLY_OPA_DISP++, 0x06, play->objectCtx.status[this->enHy.headObjIndex].segment);
-        gSegments[6] = VIRTUAL_TO_PHYSICAL(play->objectCtx.status[this->enHy.headObjIndex].segment);
-        gSegments[6] = VIRTUAL_TO_PHYSICAL(play->objectCtx.status[this->enHy.skelLowerObjIndex].segment);
+        gSegments[6] = OS_K0_TO_PHYSICAL(play->objectCtx.status[this->enHy.headObjIndex].segment);
+        gSegments[6] = OS_K0_TO_PHYSICAL(play->objectCtx.status[this->enHy.skelLowerObjIndex].segment);
```

Comment:
What's the reasoning for replacing `VIRTUAL_TO_PHYSICAL` with `OS_K0_TO_PHYSICAL`?

Reply:
`OS_K0_TO_PHYSICAL` is a real libultra macro while `VIRTUAL_TO_PHYSICAL` is something the decomp people came up with. Since both accomplish the same I think it is better to stick to the official libultra macro to avoid redundancy

Reply:
One thing that isn't great is that `SEGMEMT_TO_VIRTUAL` doesn't match with the rest. I would say it would be better to either keep the other macros, but more or less just as wrappers for the official ones, or rename this one to `SEGMENT_TO_K0` or possibly with the `OS` prefix though it isn't an official macro.

Reply:
`SEGMENT_TO_K0` sounds good. I would prefer to avoid the `OS_` prefix exactly because of it not being a libultra macro

Reply:
On a personal note I don't really like this change. It's good that we know the actual name of the macro and have identified it, but it makes the code harder to understand. I tried looking it up and still don't know what K0 means in this context, as it doesn't seem to be industry standard but specific vocabulary to libultra. 

I'm approving anyway, but if it was my call I would leave the old name and add a comment documenting the original name of the macro at the macro definition instead. I know that isn't consistent but most of libultra doesnt reach this far into the regular code of the game, this macro is everywhere.

Reply:
Might be a good idea to bring this to discord, instead of hiding it in a rather large PR that many people haven't looked through.